<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>babelView</class>
 <widget class="QWidget" name="babelView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>391</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Babel</string>
  </property>
  <widget class="QPlainTextEdit" name="toRecieve">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>371</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="toSend">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sendB">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>310</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Enviar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectB">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>150</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Conectar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="conIP">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>70</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>IP</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
